data/projects/integrate-pyflakes-AST.md

```diff
@@ -0,0 +1,24 @@
+flake8 has many plugins which are commonly used in the python development community.  flake8 is a wrapper, similar to coala, that provides that use the python-enhanced AST to its plugins.  Instead of coala using the flake8 wrapper, which adds another level of complexity and uncertainty with regards to configuration, coala would like to directly load the python-enhanced AST and invoke the flake8 plugins.
+This allows the coala and flake8 communities to cooperate on plugin development using common plugin framework.
+
+This project focuses on integrating the [`pyflakes`](https://pypi.python.org/pypi/pyflakes)-enhanced [AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree) into coala and wrap flake8 AST plugins [`flake8-future-import`](https://github.com/xZise/flake8-future-import) and [`flake8-string-format`](https://github.com/xZise/flake8-string-format) using it.
+
+This can be done by creating a meta bear like [AnnotationBear](https://github.com/coala/coala-bears/blob/1892b64722f1c10bd2a29611af4dcc18ad76af3a/bears/general/AnnotationBear.py) and [ClangBear](https://github.com/coala/coala-bears/blob/27a06d19076633a4e796472cf6d6ad55c27045c5/bears/c_languages/ClangBear.py), say PyflakesASTBear. Then use this bear to wrap the flake8 AST plugins. Tutorials for writing and testing bears can be found in our [documentation](https://api.coala.io).
+
+The flake8 plugins needed to installed can be added to a dynamically created venv. Check out [this issue](https://gitlab.com/coala/package_manager/issues/25).
+
+#### Milestones
+
+##### GSOC 2017 COMMUNITY BONDING
+
+* The participant has done prior contributions by writing and testing coala-bears. Check out [existing bear proposal](https://github.com/coala/coala-bears/issues?q=is%3Aissue+is%3Aopen+label%3A%22bear+proposal%22).
```

Comment:
In this phase either a concept or a cEP (depending on the severity of changes to be done) should be written and finished. I'd say a concept is enough. So have something like "Api concept exists"